src/widgets/Tabs/PatientSelect.js

```diff
@@ -59,10 +61,16 @@ const PatientSelectComponent = ({
   onFilterData,
   patients,
   selectPatient,
+  updateForm,
 }) => {
   const columns = React.useMemo(() => getColumns(MODALS.PATIENT_LOOKUP), []);
   const { pageTopViewContainer } = globalStyles;
   const keyboardIsOpen = useKeyboardIsOpen();
+  const debouncedFilter = useDebounce(onFilterData, 500);
```

Comment:
I tried this with `250` and for me it works fine and has a little less lag. Totally dependent on the typist though and if it triggers to much it does get a bit janky (50 is too low!!). 500 is probably good

Reply:
Compromised.. went with `300`  though the `250` was fine for me too 🤷‍♂️